client/src/app/components/stardust/TransactionPayloadState.tsx

```diff
@@ -0,0 +1,6 @@
+export interface TransactionPayloadState {
```

Comment:
This should be TransactionPayloadState.ts, not tsx

Reply:
Done.